spec/event_store/event_builder_spec.rb

```diff
@@ -102,6 +102,19 @@ module EventFramework
         end
       end
 
+      describe "system metadata" do
+        let(:event) do
+          row[:metadata]["metadata_type"] = "system"
+          row[:metadata].delete("user_id")
+          event_builder.call(row)
+        end
+
+        it "has is an UnattributedMetadata" do
```

Comment:
is this a typo? Should it say `has a SystemMetadata` ? or is system metadata similar to UnattributedMetadata?

Reply:
Very much a copy paste typo!

Reply:
Fixed